### DISCUSSION TITLE
Start assigning priority to issues?

### DISCUSSION THREAD

**Discussion by northwestwitch** [Clinical-Genomics/scout#2640] [Ideas]
Would be helpful to organize the work, especially now that the number of contributors starts to grow

**Comment by mikaell** [comment]
Yes. I support the idea. I suggest using too many heirarcys, three should be enough. For example a traffic light of red (urgent), yellow (medium) and green (low) priority.

**Reply by dnil** [reply]
You do constitute a good use case for this; otherwise I would classify it as pure overhead. 😁

**Reply by dnil** [reply]
Let's reuse the prio tags from the other CG repos then?

**Reply by northwestwitch** [reply]
Everything that is a bug is prio1 I'd say
